src/mobile-token/utils.ts

```diff
@@ -48,7 +47,24 @@ export const wipeToken = async (tokensIds: string[], traceId: string) => {
        * already removed, in this case, just wipe the local token.
        */
       if (isEntityDeletedError(err)) {
-        await mobileTokenClient.clear();
+        notifyBugsnag(Error(`Token removed from backoffice ${id}`), {
+          errorGroupHash: 'token',
+          metadata: {
+            traceId,
+            description: `Token already deleted in backoffice ${id}, start new`,
+          },
+        });
+        break; // No need to do anything, proceed to clear the local token
+      } else {
+        notifyBugsnag(err, {
+          errorGroupHash: 'token',
+          metadata: {
+            traceId,
+            description: `Other error during wipe token, throw the error`,
+          },
+        });
```

Comment:
Should this just be log to Bugsnag instead? I would think the thrown err gets notified to Bugsnag when it is catcher higher up.

Reply:
Yes, we can do that, thanks for the reminder!